addons/gamification/__openerp__.py

```diff
@@ -2,7 +2,6 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 {
     'name': 'Gamification',
-    'version': '1.0',
```

Comment:
@dreispt nope, it's just that it was set to it's <a href='https://github.com/odoo/odoo/blob/master/openerp/modules/module.py#L302'>default</a> value